src/engine/sidechain/enginesidechain.cpp

```diff
@@ -1,11 +1,3 @@
-// This class provides a way to do audio processing that does not need
-// to be executed in real-time. For example, broadcast encoding
-// and recording encoding can be done here. This class uses double-buffering
-// to increase the amount of time the CPU has to do whatever work needs to
-// be done, and that work is executed in a separate thread. (Threading
-// allows the next buffer to be filled while processing a buffer that's is
-// already full.)
-
```

Comment:
This also looks unrelated. Can you remove the unrelated changes from this PR? That would bring down the line count and make it easier to review.